.ci/doc/templates/print-result.tpl.kt

```diff
@@ -1,13 +1,13 @@
 import io.kuzzle.sdk.Kuzzle
-import io.kuzzle.sdk.protocol.WebSocket
+import io.kuzzle.sdk.protocol.Http
 import java.util.Date
 
 import java.util.concurrent.ExecutionException
 import io.kuzzle.sdk.coreClasses.lang.Lang
 import io.kuzzle.sdk.coreClasses.responses.Response
 
 fun main() {
-  val ws = WebSocket("kuzzle")
+  val ws = Http("http://kuzzle:7512")
```

Comment:
You should change the command config in `.ci/doc/config.yml` to add the protocol as environment variable
like so `cmd: SNIPPET_PROTOCOL="{{ snippet.protocol }}" /mnt/.ci/doc/test-snippet.sh {{ snippet.source }}`

And instanciate the proper protocol inside the snippet based on the environment variable `SNIPPET_PROTOCOL` instead of hardcoding the protocol to use.

You should apply this behaviour to every template